server/Windows/wf_peer.c

```diff
@@ -279,11 +279,11 @@ DWORD WINAPI wf_peer_main_loop(LPVOID lpParam)
 	if (!client->Initialize(client))
 		goto fail_client_initialize;
 
+	context = (wfPeerContext*) client->context;
+	
 	if (context->socketClose)
```

Comment:
Undefined behavior on "wfPeerContext *context", as it is accessed before being initialized or assigned. ^ Fixed